### ISSUE TITLE
Turtle Mode On Multishot

### ISSUE THREAD

**Issue by j33bs** [ButterFlight/butterflight#161]
Can we please have a mode that switches to dshot and reverses props (from multishot) specifically for helio users looking for 32k pid loop. Sorry if this issue has already been listed, i couldn't find it.

**Comment by Clynikal** [comment]
Thanks for posting this request, I think it’s a request to enable dshot on aux switch(s) for any dshot command that way we can still do beacon for failsafe in a mix.

**Comment by Zuldan** [comment]
Turtle Mode On Multishot would be fantastic. @Clynikal thank you for looking into this. We appreciate it.

**Comment by j33bs** [comment]
yeah if enabled on a switch i think their may need to be a change within the gui to allow flip over after crash to be assigned an aux channel while multishot is selected under configuration.

**Comment by monko760** [comment]
Changing esc protocol usually requires the fc to be rebooted, doe sit not? Me thinks it wouldn't be to easy, but Im dumb and just wanted to say I WANT THIS FEATURE!

**Comment by mirfpv** [comment]
upvote!

**Comment by allanragec** [comment]
upvote
